modules/widgets/eu-cookie-law.php

```diff
@@ -99,16 +99,19 @@ function __construct() {
 		 */
 		function enqueue_frontend_scripts() {
 			wp_enqueue_style( 'eu-cookie-law-style', plugins_url( 'eu-cookie-law/style.css', __FILE__ ), array(), '20170403' );
-			wp_enqueue_script(
-				'eu-cookie-law-script',
-				Assets::get_file_url_for_environment(
-					'_inc/build/widgets/eu-cookie-law/eu-cookie-law.min.js',
-					'modules/widgets/eu-cookie-law/eu-cookie-law.js'
-				),
-				array(),
-				'20180522',
-				true
-			);
+
+			if ( ! Jetpack_AMP_Support::is_amp_request() ) {
```

Comment:
Let's still add the class exists check in here, as the widget can be loaded earlier on WordPress.com.

```suggestion
			if ( ! class_exists( 'Jetpack_AMP_Support' ) || ! Jetpack_AMP_Support::is_amp_request() ) {
```

Reply:
@jeherve This has been fixed in the latest commit.